### DISCUSSION TITLE
New computer - FP interface query

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#796] [Help]
My old computer is dying, so I bought a new computer. Most apps look pretty much as I would expect. But FP's menus and toolbars are tiny. I'm including a screenshot that compares FP's menu to the one from TheBrain. At first I was thinking it was a monitor setting, but that can't be since other programs show menus and other parts of their interfaces as I would expect.

<img width="374" alt="image" src="https://user-images.githubusercontent.com/8197883/199651453-e00ba6e2-9540-4bdd-aaaa-0f2ff69e0f3c.png">

I also had to change the scale (zoom) of the mindmaps because FP makes them smaller, too.

Thoughts? Fixes?

Thanks.

~ Bal

**Comment by sifran-github** [comment]
you have a lot of settings in tools->preferences->appearance->look and feel.

**Reply by wordmuse** [reply]
Thanks. I am aware of that. But I don't want to just muck with them in the blind, risking making things worse. Especially if there are people here who _know_ what settings are responsible for the different parts of the interface.

Addendum: I realized that I'm probably too cautious - I'll play around with it and see how it goes. Thanks again.

Addendum #2: I'm playing with it, but not getting good results. My custom icons, which were perfect size on my old computer, are so small on the new computer, that I almost have to squint to see them.  For some reason the node widths, which worked fine on the old machine are now too narrow on the new machine, and the styles all have to be adjusted.  This seems like way more work than what I would expect, which leads me to think I'm doing something wrong. So any specific help would be most welcome.

~ Bal

**Comment by wordmuse** [comment]
Is there an option in the Preferences that controls the scaling of a custom-made icon (format .png)?

**Comment by wordmuse** [comment]
While I am still curious to see if Freeplane has ways to do what I asked above, I gave up. I am now resizing my custom icons so that they work with the new machine. As I have a lot of them, it's a time-consuming process, which is why I'd hoped for an interface options answer. In any event, this query can now be considered moot. 

~ Bal

**Reply by dpolivaev** [reply]
Freeplane can resize only svg icons. For the rest the user should prepare them in the required sizes. There are some batch converters that allow recalculating the whole collection of images with sizes needed but do not ask me or us, ask Google.
